plugins/gx_digital_delay.lv2/gx_digital_delay.ttl

```diff
@@ -81,6 +89,8 @@ rdfs:comment """
         lv2:default 1.2e+02 ;
```

Comment:
you changed the plugin below to remove scientific notation, but not on this one.
No biggie, but would be nice to be consistent.